selfdrive/global_ti.py

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env python3
```

Comment:
i dont see the need for this. unless i am not seeing something

Reply:
Due to the reasons MoreTore/RyleyMCC gave (which I can't see for some reason by the way, I just see it in a screenshot he sent me), I think it's best that we just leave this as it is, along with car_helpers.py. 

It's an extra file, yes, but this works fine and it's not certain if it would reliably work otherwise.

Reply:
thanks for the explanation.